README.md

```diff
@@ -307,10 +306,10 @@ final class GitHubSearchViewController: UIViewController {
         super.viewDidLoad()
 
         searchBar.rx.text
-            .bind(to: viewStream.input.accept(for: \.searchText))
+            .bind(to: viewStream.input.searchText)
             .disposed(by: disposeBag)
 
-        viewStream.output.observable(for: \.repositories)
+        viewStream.outpu.repositories
```

Comment:
typo 👀

Reply:
Thank you.
I've fixed it. 650c5025b78f877e57143c0ea4262a068e56bc53